modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -763,6 +765,37 @@ public abstract class Base${schemaName}ResourceImpl
 				throw new UnsupportedOperationException("This method needs to be implemented");
 			</#if>
 		}
+
+		<#if (parserMethodDataTypes?has_content)>
+
+			<#assign parserMethodDataTypesSimplified = freeMarkerTool.sequenceDistinct(parserMethodDataTypes) />
+
+			<#list parserMethodDataTypesSimplified as parserMethodDataType>
```

Comment:
You can do it inline:

`<#list freeMarkerTool.sequenceDistinct(parserMethodDataTypes) as parserMethodDataType>`